examples/MomentoApplication/Program.cs

```diff
@@ -6,43 +6,94 @@
 using Momento.Sdk.Exceptions;
 using Momento.Sdk.Responses;
 
-namespace MomentoApplication
+string MOMENTO_AUTH_TOKEN = Environment.GetEnvironmentVariable("MOMENTO_AUTH_TOKEN");
+// Use a GUID for the cache name so it's unlikely to exist already.
+string CACHE_NAME = Guid.NewGuid().ToString();
```

Comment:
do we want to use `readonly string` instead per Kenny's comment?
https://www.notion.so/momentohq/Kenny-notes-cf5dc7b645a34cb094b7ffee69998624#6b61a3c6e2404b3e8593470f1a5260ae

Reply:
👍

Reply:
I don't think we should uuid here, because we have a default limit of ~5-10 caches per user.  We don't want folks who run this program a few times to end up hitting their limit.  In fact I think I might be inclined to say we should just use the same default cache name here that you get when you first run the momento cli.